gadgets/matlab/CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
-include_directories(${MATLAB_INCLUDE_DIR})
+include_directories(
+    ${MATLAB_INCLUDE_DIR}
+    ${ACE_INCLUDE_DIR}
```

Comment:
You should only need MATLAB here. The rest should be imported automatically by linking against the appropriate libraries